modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -2860,7 +2860,7 @@ private void ackMemoryConfiguration() {
             return;
 
         U.log(log, "System cache's DataRegion size is configured to " +
-            (memCfg.getSystemRegionInitialSize() / (1024 * 1024)) + " MB. " +
+            (memCfg.getSystemDataRegionConfiguration().getInitialSize() / (1024 * 1024)) + " MB. " +
```

Comment:
Is it NPE safe? It will be greate to check this with a test.

Reply:
I think this is a border where we do not need to check NPE. Otherwise, we need to check other similar properties too like `org.apache.ignite.configuration.DataStorageConfiguration#getDefaultDataRegionConfiguration`.